### PR TITLE
Fixes for ICP-13200, ICP-13202

### PR DESCRIPTION
**Pull request by KKlimczukS**
- Removed unnecessary calls of meterGet.
- Fixes for setting device preferences (ICP-13200, ICP-13202)
- Removed boolRange boilerplate code - since device settings of that type are not exposed in those DTHs
@greens @dkirker Please, could You take a look at the code ?

cc: @MWierzbinskaS @ZWozniakS @PKacprowiczS @BJanuszS @MGoralczykS

### CODE REVIEW

File: devicetypes/fibargroup/fibaro-walli-double-switch.src/fibaro-walli-double-switch.groovy [hunk 1]
```diff
@@ -196,12 +166,12 @@ private getCommandValue(preference) {
 	def parameterKey = preference.key
 	switch (preference.type) {
 		case "boolean":
-			return settings."$parameterKey" ? preference.optionActive : preference.optionInactive
-		case "boolRange":
-			def parameterKeyBoolean = parameterKey + "Boolean"
-			return settings."$parameterKeyBoolean" ? settings."$parameterKey" : preference.disableValue
+            // boolean values are returned as strings from the UI preferences
+            return settings."$parameterKey" == 'true' ? preference.optionActive : preference.optionInactive
+        case "range":
```
Comment (greens) [inline devicetypes/fibargroup/fibaro-walli-double-switch.src/fibaro-walli-double-switch.groovy]: nit: spacing

File: devicetypes/fibargroup/fibaro-walli-double-switch.src/fibaro-walli-double-switch.groovy [hunk 2]
```diff
@@ -381,10 +345,7 @@ def childOnOff(deviceNetworkId, value) {
 private onOffCmd(value, endpoint = 1) {
 	delayBetween([
 		encap(zwave.basicV1.basicSet(value: value), endpoint),
-		encap(zwave.basicV1.basicGet(), endpoint),
-		"delay 3000",
-		encap(zwave.meterV3.meterGet(scale: 0), endpoint),
-		encap(zwave.meterV3.meterGet(scale: 2), endpoint)
```
Comment (greens) [inline devicetypes/fibargroup/fibaro-walli-double-switch.src/fibaro-walli-double-switch.groovy:387]: were these meter gets causing problems? Does the device update us automatically?

Comment (KKlimczukS) [reply devicetypes/fibargroup/fibaro-walli-double-switch.src/fibaro-walli-double-switch.groovy:387]: Device updates automatically, it was noticed that those additional meterGets choke that device sometimes.

File: devicetypes/fibargroup/fibaro-walli-double-switch.src/fibaro-walli-double-switch.groovy (hunk 1 above)
Comment (KKlimczukS) [reply devicetypes/fibargroup/fibaro-walli-double-switch.src/fibaro-walli-double-switch.groovy]: Fixed.

**Review by greens — APPROVED** [review]

**Pull request merged**